stable/oauth2-proxy/templates/secret.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.config }}
```

Comment:
this will basically always match if you do not override you values file with:
`config: {}`
is that intentional?

Reply:
It was intentional. I did not want to have people who are already using this behaviour impacted.

For those of us who manage secrets outside of git commits to chart overrides, then we can simply set the config: {} and continue on.

Reply:
There's also `config.configFile`. With that logic you'd always create the secret if someone specifies a config file.

Common practice is to add an `existingSecret` field.

Reply:
Nice, forgot about existingSecret. Will update m